x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/multiple_conditions.cy.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EXCEPTION_ITEM_ACTIONS_BUTTON, REMOVE_EXCEPTION_BTN } from '../../../screens/rule_details';
+import { deleteAlertsAndRules } from '../../../tasks/common';
+import { getNewRule } from '../../../objects/rule';
+
+import { createRule } from '../../../tasks/api_calls/rules';
+import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+import { esArchiverResetKibana, esArchiverUnload } from '../../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+import {
+  openExceptionFlyoutFromEmptyViewerPrompt,
+  goToExceptionsTab,
+  removeException,
+} from '../../../tasks/rule_details';
+import {
+  addExceptionEntryFieldValue,
+  addExceptionEntryFieldValueValue,
+  addExceptionFlyoutItemName,
+  submitNewExceptionItem,
+} from '../../../tasks/exceptions';
+import {
+  ADD_AND_BTN,
+  ADD_OR_BTN,
+  EXCEPTION_CARD_ITEM_NAME,
+  EXCEPTION_CARD_ITEM_CONDITIONS,
+  EXCEPTION_ITEM_VIEWER_CONTAINER,
+} from '../../../screens/exceptions';
+
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
+
+describe(
+  'Add multiple conditions and validate the generated exceptions',
+  { testIsolation: false },
+  () => {
+    before(() => {
+      esArchiverResetKibana();
```

Comment:
Please, check if the use of this method is extremely necessary. Thanks.

Reply:
Yes, i removed deleteRulesAndAlerts as esArchiverResetKibana will be enough here